docs/user-guide/sdks-using.md

```diff
@@ -40,13 +42,17 @@ If you download the Python SDK packages from Zowe.org, the downloaded folder con
 pip install <fileName>.whl
 ```
 
+### Java
+
+Requires Java runtime version 11 and above.  
+
 ## Getting started
 
 To get started, import the SDK packages to your project. You can pull the packages from an online registry, or download the packages from Zowe.org to install locally.
 
-### Install SDK from online registry
+### Install Node.js and Pyhton SDK from online registry
 
-Pull the packages from an online registry such as npm or PyPi.
+Pull the packages from an online registry such as npm or PyPi or maven.
```

Comment:
If we plan to have another header for Java, and make this one specific to Node and Python, I think we should leave Maven out of this list 😋

Reply:
I removed it. It was left there in my branch always meant to clean it up. See latest commit. thanks